db/hybrid_logical_vector.go

```diff
@@ -952,6 +952,15 @@ func DefaultLWWConflictResolutionType(ctx context.Context, conflict Conflict) (B
 	if conflict.LocalHLV == nil || conflict.RemoteHLV == nil {
 		return nil, errors.New("local or incoming document is nil for resolveConflict")
 	}
+	localDeleted, _ := conflict.LocalDocument[BodyDeleted].(bool)
+	remoteDeleted, _ := conflict.RemoteDocument[BodyDeleted].(bool)
```

Comment:
nit:

I think I'd be inclined to create a `isDeleted` function on `Body` which has a lot of methods like `ExtractDeleted` and `getExpiry`